product_available_by_warehouse/models/product.py

```diff
@@ -40,3 +40,11 @@ def get_product_available_by_warehouse(self):
                 'outgoing_qty': outgoing_qty,
             }
         return res
+
+    @api.multi
+    def _get_domain_locations(self):
```

Comment:
Hi @JulioSerna 

I do not agree with the super, it is a risk change that can affect other features, maybe should do the computing independent of the context to avoid the change in a common use function like `_get_domain_locations`

But to be honest, actually i understand that qty_available only show product quantities on internal locations...  i don't know if i'm wrong, but will useful that show a case (with a video or screenshot) when qty available, sum from a not internal location